cubids/cubids.py

```diff
@@ -390,6 +485,7 @@ def change_filename(self, filepath, entities):
         sub_ses = sub + "_" + ses
 
         if "run" in list(entities.keys()) and "run-0" in filepath:
+            # XXX: This adds an extra leading zero to run.
```

Comment:
I'm not sure we want this hard coded. wdyt?

Reply:
I have a bug report issue open about it (#272), but I didn't feel confident tackling it in this PR.